src/components/Accessibility.js

```diff
@@ -2,10 +2,11 @@
 
 import type {Node as ReactNode} from "React";
 import React from "react";
-import icons from "../icons";
+/*import icons from "../icons";*/
```

Comment:
Generally we try to remove unused code rather than comment it out. Otherwise the code base can quickly fill up with commented out code. If we ever need to use some some old code again we can look in the git history to find it.